src/pages/rekey-search/rekey-search.html

```diff
@@ -73,8 +73,8 @@ <h3>
       </p>
     </div>
 
-    <p class="results-message" *ngIf="searchResults.length > 0">
-      Displaying <span>{{searchResults.length}}</span> results
+    <p class="results-message" *ngIf="!isBookedTestSlotEmpty(pageState.bookedTestSlot$ | async)">
+      <test-slot [slot]="(pageState.bookedTestSlot$ | async)" [hasSlotChanged]="false" [showLocation]="false"></test-slot>
```

Comment:
Just out of curiosity, how does `<test-slot>` know to display the 'Rekey' button?

Reply:
Well not exactly, it displays Start test if the slot is booked for today. And Rekey if yesterday, or earlier. So will have to work on that bit as well. (Would like to keep a flow of PRs rather than standing on one of them for a long time)
Making a note of this.

Reply:
Was just curious, sounds like its handled elsewhere 👍